packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,471 @@
+                    position: relative;
+                    left: 2px;
+                }
+            }
+        }
+        .aa-InputWrapper {
+            position: relative;
+            width: 100%;
+            order: 3;
+            // input of the searchbox, where the placeholder and query appear
+            input.aa-Input {
+                width: 100%;
+                height: calc(var(--aa-spacing) * 2.5);
+                appearance: none;
+                background: none;
+                border: none;
+                font: inherit;
+                color: var(--aa-text-color);
+                // remove native appearence
+                &::-webkit-search-decoration,
+                &::-webkit-search-cancel-button,
+                &::-webkit-search-results-button,
+                &::-webkit-search-results-decoration {
+                    -webkit-appearance: none;
+                }
+                &::placeholder {
+                    opacity: 1;
+                    color: var(--aa-muted-color);
+                }
+                // remove focus effect as we moved it to parent wrapper
+                &:focus {
+                    border-color: none;
+                    box-shadow: none;
+                    outline: none;
+                }
+            }
+        }
+        .aa-InputWrapperSuffix {
+            order: 4;
+            display: flex;
+            align-items: center;
+            // accelerator to clear the query
+            button.aa-ResetButton {
+                display: flex;
+                align-items: center;
+                border: 0;
+                cursor: pointer;
+                background: none;
+                color: var(--aa-muted-color);
+                &[hidden] {
+                    display: none;
+                }
+                &:hover {
+                    color: var(--aa-text-color);
+                }
+                svg {
+                    width: var(--aa-icon-size);
+                    stroke-width: var(--aa-icon-stroke-width);
+                }
+            }
+        }
+    }
 }
 
-.aa-Label,
-.aa-TouchSearchButtonIcon {
-  align-items: center;
-  color: #777;
-  cursor: initial;
-  display: flex;
-  height: 100%;
+// ----------------
+// Panel
+// ----------------
+.aa-Panel {
+    position: absolute;
+    //reset
+    * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+    button {
+        appearance: none;
+        background: none;
+        border: none;
+    }
+    &.aa-Panel--desktop {
+        .aa-PanelLayout {
+            //display: flex;
+            margin-top: var(--aa-spacing-half);
+            padding-top: var(--aa-spacing-half);
+            border-radius: 3px;
+            background-color: var(--aa-background-color);
+            box-shadow: var(--aa-panel-shadow);
+            text-align: left;
+            padding-bottom: var(--aa-spacing-half);
+            overflow: hidden;
+            .col-results {
```

Comment:
Do we want this to be part of our theme? If so, perhaps something like `.aa-PanelLayoutResults` is more appropriate?